packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -412,13 +412,13 @@ export function processAttachMessageGCData(
 /**
  * @internal
  */
-export class TelemetryContext implements ITelemetryContext {
-	private readonly telemetry = new Map<string, TelemetryBaseEventPropertyType>();
+export class TelemetryContext implements ITelemetryContextExt {
```

Comment:
We should keep implementing both interfaces, so we make sure that the class doesn't accidentally stops being compatible with `ITelemetryContext`, since we're still passing it in code paths that receive `ITelemetryContext`. This should be doable without adding any more methods, if the `value(s)` parameter to `set()` and `setMultiple()` takes `TelemetryEventPropertyTypeExt | TelemetryBaseEventPropertyType`.
```suggestion
export class TelemetryContext implements ITelemetryContext, ITelemetryContextExt {
```